### PR TITLE
Move care and feeding docs out of installation section [SUITE-1846]

### PR DESCRIPTION
**Pull request by jodygarnett**
In the installation sections, there are pages on "working with" the install. It makes more sense to move it out of there and into the sysadmin section.

### CODE REVIEW

File: docs/usermanual/source/sysadmin/redhat/index.rst [hunk 1]
```diff
@@ -1,4 +1,262 @@
...
+      <Context displayName="geoserver"
+               docBase="/usr/share/boundless/geoserver"
+               path="/geoserver"/>
+
+#. Create the :file:`geowebcache.xml` with the following content:
+   
+   .. code-block:: xml
+   
+      <Context displayName="geowebcache"
+               docBase="/usr/share/boundless/geowebcache"
```
Comment (jodygarnett) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: Is this the correct package install location @nstires-boundless ?

File: docs/usermanual/source/sysadmin/ubuntu/index.rst [hunk 2]
```diff
@@ -1,4 +1,247 @@
...
+   .. note:: Make sure the line is uncommented (does not start with ``#``).
+
+#. Save and close the file.
+
+#. Restart Tomcat.
+
+Using Boundless Suite with custom Tomcat
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Boundless Suite packages can be used to manage the contents :file:`/usr/share/opengeo` components while making use of your own Tomcat application server.
```
Comment (jodygarnett) [inline docs/usermanual/source/sysadmin/ubuntu/index.rst]: Is this the correct install location @nstires-boundless ?

File: docs/usermanual/source/sysadmin/redhat/index.rst [hunk 3]
```diff
@@ -1,4 +1,262 @@
...
+.. note:: The Tomcat service used by Boundless Suite is pulled in from standard repository sources, and is not specific to Boundless Suite.
+
+Controlling the Tomcat service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To start/stop/restart the Tomcat service:
+
+  .. code-block:: bash
+ 
+     service tomcat start|stop|restart
```
Comment (nstires-boundless) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: `service tomcat8 start|stop|restart`

File: docs/usermanual/source/sysadmin/redhat/index.rst [hunk 4]
```diff
@@ -1,4 +1,262 @@
...
+
+.. note:: Depending on the distribution and version the service name may be one of "tomcat", "tomcat5", or "tomcat6". Use the :command:`service` command to determine which one is installed:
```
Comment (nstires-boundless) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: Assuming they're using our packaging, this line is OBE.

File: docs/usermanual/source/sysadmin/redhat/index.rst [hunk 5]
```diff
@@ -1,4 +1,262 @@
...
+
+         service --status-all | grep tomcat
+
+#. Search for "8080" (around line 75) and change the ``port`` attribute to the desired value.
+
+#. Restart tomcat. 
+
+   .. code-block:: bash
+
+        service tomcat restart
```
Comment (nstires-boundless) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: `service tomcat8 restart`

File: docs/usermanual/source/sysadmin/redhat/index.rst [hunk 6]
```diff
@@ -1,4 +1,262 @@
...
+--------------------------
+
+The Tomcat and PostgreSQL services run on ports **8080** and **5432** respectively. These ports can often conflict with existing services on the systemk, in which case the ports must be changed. 
+
+Changing the Tomcat port
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+To change the Tomcat port:
+
+#. Edit the file :file:`/etc/tomcat/server.xml`.
```
Comment (nstires-boundless) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: `/etc/tomcat8/server.xml`

File: docs/usermanual/source/sysadmin/redhat/index.rst [hunk 7]
```diff
@@ -1,4 +1,262 @@
...
+The Tomcat and PostgreSQL services run on ports **8080** and **5432** respectively. These ports can often conflict with existing services on the systemk, in which case the ports must be changed. 
+
+Changing the Tomcat port
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+To change the Tomcat port:
+
+#. Edit the file :file:`/etc/tomcat/server.xml`. 
+
+   .. note:: Depending on the distribution and version replace "tomcat" with "tomcat5" or "tomact6" accordingly. Use the :command:`service` command to determine which one is installed:
```
Comment (nstires-boundless) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: Assuming they're using our packaging, this line is OBE.

File: docs/usermanual/source/sysadmin/redhat/index.rst [hunk 8]
```diff
@@ -1,4 +1,262 @@
...
+-------------------
+
+Changing the Tomcat Java
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you wish to use the Oracle Java 7 JRE (rather than the OpenJDK 7 installed by default):
+
+#. Download and install Oracle Java 7 JRE.
+
+#. Open :file:`/etc/sysconfig/tomcat` and update the ``JAVA_HOME`` environment variable.
```
Comment (nstires-boundless) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: `/etc/sysconfig/tomcat8`

File: docs/usermanual/source/sysadmin/redhat/index.rst [hunk 9]
```diff
@@ -1,4 +1,262 @@
...
+
+   .. note:: Make sure the line is uncommented (does not start with ``#``).
+
+#. Save and close the file.
+
+#. Restart Tomcat.
+
+Using Boundless Suite with custom Tomcat
```
Comment (nstires-boundless) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: Entire block through line 160 is OBE, part of packages. Only needed for WAR install.

File: docs/usermanual/source/sysadmin/redhat/index.rst [hunk 10]
```diff
@@ -1,4 +1,262 @@
...
+               path="/docs"/>
+
+#. Restart Tomcat.
+
+Adding other system parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can add other system or application-specific parameters that will be picked up upon restart.
+
+#. Open :file:`/etc/sysconfig/tomcat` in a text editor.
```
Comment (nstires-boundless) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: I would direct them to use the mechanism put in place within `/etc/tomcat8/suite-opts/`

File: docs/usermanual/source/sysadmin/redhat/index.rst [hunk 11]
```diff
@@ -1,4 +1,262 @@
...
+----------------------
+
+GeoServer Data Directory
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The **GeoServer Data Directory** is the location on the file system where GeoServer stores all of its configuration, and (optionally) file-based data. By default, this directory is located at: :file:`/var/lib/boundless/geoserver`. 
+
+To point GeoServer to an alternate location:
+
+#. Edit the file :file:`geoserver/WEB-INF/web.xml`.
```
Comment (nstires-boundless) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: This is set in `/etc/tomcat8/suite-opts/geoserver`

File: docs/usermanual/source/sysadmin/ubuntu/index.rst [hunk 12]
```diff
@@ -1,4 +1,247 @@
 .. _sysadmin.ubuntu:
 
 Ubuntu Suite Administration
```
Comment (nstires-boundless) [inline docs/usermanual/source/sysadmin/ubuntu/index.rst]: I can't speak towards the DB side, but the tomcat and geoserver administration is the same as centos.

File: docs/usermanual/source/sysadmin/ubuntu/index.rst (hunk 2 above)
Comment (nstires-boundless) [inline docs/usermanual/source/sysadmin/ubuntu/index.rst]: Should be `/opt/boundless/suite/geoserver/`

File: docs/usermanual/source/sysadmin/redhat/index.rst (hunk 1 above)
Comment (nstires-boundless) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: Format is `/opt/boundless/suite/<product>`

File: docs/usermanual/source/sysadmin/redhat/index.rst (hunk 4 above)
Comment (jodygarnett) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: What is OBE, changing the text to read:

````
.. note:: Depending on the distribution and version the service name may be one of "tomcat", "tomcat8", or "tomcat9". Use the :command:`service` command to determine which one is installed:
````

Comment (nstires-boundless) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: The packages only use one flavor of tomcat, tomcat8. So I don't think we need to add this.

File: docs/usermanual/source/sysadmin/redhat/index.rst (hunk 10 above)
Comment (jodygarnett) [inline docs/usermanual/source/sysadmin/redhat/index.rst]: Yeah I am on it ...

**Pull request merged**